website/content/docs/enterprise/license/utilization-reporting.mdx

```diff
@@ -0,0 +1,253 @@
+---
+layout: docs
+page_title: Automated license utilization reporting
+description: >-
+  Learn what data HashiCorp collects to meter Enterprise license utilization. Enable or disable reporting. Review sample payloads and logs.
+---
+
+# Automated License utilization reporting
+
+Automated license utilization reporting sends license utilization data to
+HashiCorp without requiring you to manually collect and report them. It also
+lets you review your license usage with the monitoring solution you already use
+(for example Splunk, Datadog, or others) so you can optimize and manage your
+deployments. Use these reports to understand how much more you can deploy under
+your current contract, protect against overutilization, and budget for predicted
+consumption.  
+
+Automated reporting shares the minimum data required to validate license
+utilization as defined in our contracts. They consist of mostly computed metrics
+and will never contain Personal Identifiable Information (PII) or other
+sensitive information. Automated reporting shares the data with HashiCorp using
+a secure, unidirectional HTTPS API and makes an auditable record in the product
+logs each time it submits a report.
+
+## Enable automated reporting
+
+To enable automated reporting, you need to make sure that outbound network
+traffic is configured correctly and upgrade your enterprise product to a version
+that supports it. If your installation is air-gapped or network settings are not
+in place, automated reporting will not work. 
+
+### 1. Allow outbound HTTPS traffic on port 443
+
+Make sure that your network allows HTTPS egress on port 443 from
+https://reporting.hashicorp.services by allow-listing the following IP
+addresses: 
+
+- 100.20.70.12
+- 35.166.5.222
+- 23.95.85.111
+- 44.215.244.1
+
+### 2. Upgrade
+
+Upgrade to a release that supports entitlement utilization reporting. These
+releases include:
+
+- [Vault Enterprise 1.14.0](https://releases.hashicorp.com/vault/) and later
+
+
+### 3. Check logs
+
+Automatic license utilization reporting will start sending data within 24 hours.
+Check the product logs for records that the data sent successfully.
+
+Instructions on how to check logs. 
```

Comment:
Should we add specific guidance here? If so: `grep reporting <path to vault log>`

I think it would also be okay to give the same guidance as [below](https://github.com/hashicorp/vault/pull/20898/files#diff-13490f1ee5e6b5cbb1370b297aeba8eb2c97815681b4479853631d2326b7b606R144). Either way works for me. 😄

Reply:
Ops, that was a self-reminder note 🫣  Thank you for catching!